api/GPUSupportedFeatures.json

```diff
@@ -113,6 +113,825 @@
+              "version_added": "120"
+            },
+            "chrome_android": {
+              "version_added": "121"
+            },
+            "deno": {
+              "version_added": false
+            },
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": "preview"
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror",
+            "webview_ios": "mirror"
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "feature_texture-compression-astc": {
+        "__compat": {
+          "description": "<code>texture-compression-astc</code> feature",
+          "spec_url": "https://gpuweb.github.io/gpuweb/#texture-compression-astc",
+          "tags": [
+            "web-features:webgpu"
+          ],
+          "support": {
+            "chrome": {
+              "version_added": "113"
+            },
+            "chrome_android": {
+              "version_added": "121"
+            },
+            "deno": [
+              {
+                "version_added": "1.39",
+                "flags": [
+                  {
+                    "type": "runtime_flag",
+                    "name": "--unstable-webgpu"
+                  }
+                ]
+              },
+              {
+                "version_added": "1.8",
+                "version_removed": "1.32"
+              }
+            ],
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": "preview"
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror",
+            "webview_ios": "mirror"
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "feature_texture-compression-astc-sliced-3d": {
+        "__compat": {
+          "description": "<code>texture-compression-astc-sliced-3d</code> feature",
+          "spec_url": "https://gpuweb.github.io/gpuweb/#texture-compression-astc-sliced-3d",
+          "tags": [
+            "web-features:webgpu"
+          ],
+          "support": {
```

Comment:
It's not supported in Chrome yet. Only in the spec.

Reply:
Ah, OK. BCD doesn't let you include features not supported anywhere, so I've deleted this data point for now. I'll add notes to the corresponding content PR to make it clear that these currently aren't supported anywhere.